src/Mount.cpp

```diff
@@ -3740,3 +3884,33 @@ void Mount::testUART_vactual(TMC2209Stepper *driver, int _speed, int _duration)
 }
     #endif
 #endif
+
+/////////////////////////////////
+//
+// checkRALimit
+//
+/////////////////////////////////
+void Mount::checkRALimit()
+{
+    // Check tracking limits every 5 seconds
+    if (millis() - _lastTRKCheck > 5000)
+    {
+        float trackedHours       = (_stepperTRK->currentPosition() / _trackingSpeed) / 3600.0F;  // steps / steps/s / 3600 = hours
```

Comment:
`trackedHours`, `homeRA`, and `RALimit` can all be `const` :smile: 
(IMO `const float RALimit` instead of `float const RALimit` is the common pattern)